CHANGELOG.md

```diff
@@ -4,10 +4,11 @@ All notable changes to this project will be documented in this file. This change
 
 ## [Unreleased]
 ### Changed
-- Renamed indexes-of to offsets-of.
+- Renamed `indexes-of` to `offsets-of`.
+- Renamed `get-intersection` to `get-intersecting`.
```

Comment:
is there some best practice for doing this?
checked keepachangelog.com but i didn't see any recommendations